src/App.js

```diff
@@ -9,28 +9,40 @@ import TodoListItem from './TodoListItem';
 
```

Comment:
You still gotta rename the file todoListItem.js to TodoListItem.js otherwise it breaks when I try to run it.

Reply:
On what line are you seeing a lower case todoListItem.js? I used the vsCode search and cant find it lower case anyware in my file. every one shows TodoListItem.js

Reply:
Its not in the code its your src/todoListItem.js file.

Reply:
On GitHub it was todoListItem, but on mine it was TodoListItem (capitalized) but when pushing to GitHub or comitting it wasn't renaming the file on the remote

Reply:
Hmm... you could try renaming it to something else, push, then rename it and push again.

Reply:
I had to go to GitHub and change it in the edit section, pull it into my local machine then push it back to GitHub because pushing from vsCode to GitHub wasn't fixing it.